src/Microsoft.DotNet.Build.Tasks/PackageFiles/resources.targets

```diff
@@ -20,20 +20,25 @@
           GenerateResourcesSource;
           $(CompileDependsOn);
       </CompileDependsOn>
+      <ResourcesFolderPath Condition="'$(BuildingUAPVertical)' == 'true'">$(RuntimePath)resw</ResourcesFolderPath>
```

Comment:
I thought you said that you would need this property in a different targets file. If so, this should probably be in Build.Common.props instead